pkg/server/handler/auth_test.go

```diff
@@ -1164,5 +1164,28 @@ func TestPasswordHandlerWithProvider(t *testing.T) {
 			So(resp.Code, ShouldEqual, 200)
 		})
 
+		Convey("change password success, without user", func() {
```

Comment:
It is actually better to return `profile: null` than missing the key.

“without user record”